include/shisen_cpp/camera/node/camera_node.hpp

```diff
@@ -21,10 +21,10 @@
 #ifndef SHISEN_CPP__CAMERA__NODE__CAMERA_NODE_HPP_
 #define SHISEN_CPP__CAMERA__NODE__CAMERA_NODE_HPP_
 
-#include <rclcpp/rclcpp.hpp>
-#include <shisen_cpp/camera/provider/camera_config_provider.hpp>
-#include <shisen_cpp/camera/provider/image_provider.hpp>
-#include <shisen_cpp/utility.hpp>
+#include "rclcpp/rclcpp.hpp"
```

Comment:
https://www.learncpp.com/cpp-tutorial/header-files/#includemethod

```suggestion
#include <rclcpp/rclcpp.hpp>
```